src/SFA.DAS.EmployerAccounts/Queries/GetApprenticeship/GetApprenticeshipsResponse.cs

```diff
@@ -6,5 +6,6 @@ namespace SFA.DAS.EmployerAccounts.Queries.GetApprenticeship
     public class GetApprenticeshipsResponse
     {
         public IEnumerable<Apprenticeship> Apprenticeships { get; set; }
+        public bool HasFailed { get; set; }
```

Comment:
If this is to be common across all response types then it should be put into a base class.